documentation/book/cluster-operator.adoc

```diff
@@ -495,7 +495,7 @@ See <<type-EphemeralStorage>> and <<type-PersistentClaimStorage>> for further de
 
 ===== Metrics
 
-{ProductName} uses the [Prometheus JMX exporter](https://github.com/prometheus/jmx_exporter) in order to expose metrics on each node. 
+{ProductName} uses the https://github.com/prometheus/jmx_exporter[Prometheus JMX exporter] in order to expose metrics on each node. 
```

Comment:
Instructions for styling links : http://strimzi.io/contributing/guide/#styling_links